subsys/bluetooth/host/audio/chan.c

```diff
@@ -1141,6 +1143,7 @@ void bt_audio_chan_cb_register(struct bt_audio_chan *chan, struct bt_audio_chan_
 	chan->ops = ops;
 }
 
+#if defined(CONFIG_BT_AUDIO_BROADCAST)
```

Comment:
Are we getting near the point where the chan.c should be split into broadcast and unicast?

Reply:
Yes, that should technically be possible after this PR. I'll try to remember it for a new PR once this has been merged. 

At some point, once implemented upstream in ISO, there will also be a split between broadcast sink and broadcast source